swagger.yaml

```
@@ -5568,5 +5832,82 @@ components:
             { "date": 1612553884, "calls": 89794, "endpoint": "block" },
           ]
 
+    nutlink_address:
+      type: object
+      properties:
+        metadata_url:
+          type: string
+          description: URL do specific metadata file
+          example: "https://nut.link/metadata.json"
+        metadata_hash:
+          type: string
+          description: Hash of the metadata file
+          example: "6bf124f217d0e5a0a8adb1dbd8540e1334280d49ab861127868339f43b3948af"
+        metadata:
+          type: object
+          description: The cached metadata of the `metadata_url` file.
+          additionalProperties: true
+          nullable: true
+      required:
+        - metadata_url
+        - metadata_hash
+
+    nutlink_address_tickers:
+      type: array
+      items:
+        type: object
+        properties:
+          name:
+            type: string
+            description: Name of the ticker
+          count:
+            type: integer
+            description: Number of ticker records
+          latest:
+            type: string
+            description: Block height of the latest record
+        required:
+          - name
+          - count
+          - latest
+      example:
+        - asset: "ADAUSD"
+          count: 1980038
+          latest: "2657092"
+        - asset: "ADAEUR"
+          count: 1980038
+          latest: "2657092"
+        - asset: "ADABTC"
+          count: 1980038
+          latest: "2657092"
+
+    nutlink_ticker_addresses:
+      type: array
+      items:
+        type: object
+        properties:
+          address:
+            type: string
+            description: Address of a metadata oracle
+            example: "addr_test1qpmtp5t0t5y6cqkaz7rfsyrx7mld77kpvksgkwm0p7en7qum7a589n30e80tclzrrnj8qr4qvzj6al0vpgtnmrkkksnqd8upj0"
+          latest:
+            type: string
+            description: Block height of the latest record
+            example: "2657092"
+        required:
+          - address
+          - latest
+
+    nutlink_address_ticker:
+      type: object
+      additionalProperties: true
```

```suggestion
      additionalProperties: true
```
Same as with the other addProps - is this needed?

---

Ok, seems to be needed as without explicitly specifying the object structure, the schema won't validate.